pkg/client/client_test.go

```diff
@@ -594,6 +594,45 @@ func testReadOnlyMode(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+
+func testInetFunctionsFullAndSimplifiedQuery(t *testing.T) {
+
+	url := fmt.Sprintf("postgres://%s@%s:%s/%s?sslmode=disable", serverUser, serverHost, serverPort, serverDatabase)
+	client, _ := NewFromUrl(url, nil)
+
+
+	// Normally, client.Info() result should include columns for the values returned by inet_* function calls
+
+	info, _ := client.Info()
```

Comment:
you dont need to test the regular info call, its already done in the `testInfo` function. You could move all your new code in there and wrap it in `t.Run('...')` like its done in other tests.